src/tools/sdk/index.md

```diff
@@ -38,15 +38,6 @@ A package manager can help you easily install and update the Dart SDK.
 {% include_relative _mac.md %}
 </div>
 
-Alternatives:
-
-- Not using a package manager for your OS? No problem!
-  [Download](/install/archive)
-  zip files of the Dart SDK and docs.
-- Looking for an older version?
-  Check out our [zip file archive](/install/archive) for
-  previous versions of the Dart SDK.
-
 ## What's in the SDK
 
 The SDK directory includes a `lib` directory for the
```

Comment:
SDK directory -> Dart SDK

It seems weird to talk about an SDK directory, especially when some package managers put stuff all over the place. E.g. my copy of dart is in ~/homebrew/bin, which links to ../Cellar/dart/2.0.0-dev.43.0/bin/dart, which has `bin` and `libexec` directories (the latter of which contains `bin` and `lib` directories).  I considered dropping the mentions of `bin` and `lib`, but on second thought they're kind of handy.

Reply:
In general, many occurrences of "SDK" should perhaps be "Dart SDK". Some occurrences of "SDK" could perhaps be deleted ( e.g. "About release ~SDK~ channels and version strings").